src/main/java/codechicken/nei/LayoutManager.java

```diff
@@ -176,6 +177,7 @@ public boolean isAllowedGuiAutoSearchFocus(GuiContainer gui) {
                 return true;
             }
         }
+        NEIClientConfig.logger.info("Checking for auto focus on " + guiClassName + " had no match");
```

Comment:
It's a bad idea to spam in log because you will rarely or never configure the autofocus

Reply:
Super useful for someone making a pack though, should it be configurable or just wipe entirely and let them figure it out?